git-coco

```diff
@@ -164,6 +164,47 @@ pr() {
     echo "https://console.aws.amazon.com/codesuite/codecommit/repositories/${repo}/pull-requests/${id}/changes"
 }
 
+prlist() {
+    local repo list_of_prs pull_info
+    local dst_branch src_branch is_merge pull_id author title
+
+    repo="$1"
+
+    if [ -z "$repo" ]; then
+        fail "Usage: git coco prlist <repo>"
+    fi
+
+    local header="%-10s %-60s %-8s %-100s\n"
+    local format="%-10s %-60s %-8s %s <- %s (%s)\n"
+
+    # Retrieve list of PRs for specified repo
+    local list_of_prs="$(aws_cli codecommit list-pull-requests \
+        --repository-name "$repo" \
+        --pull-request-status OPEN \
+        --query="pullRequestIds" \
+        --output text)"
+    printf "$header" "PR #" "Title" "Merge" "Branch"
+
+    # Get and parse information for each PR
+    for pull_request in $list_of_prs; do
+        local pull_info=$(aws_cli codecommit get-pull-request \
+        --pull-request-id "$pull_request" \
+        --query "pullRequest.{PR: pullRequestId, Title: title, Merge: pullRequestTargets[0].mergeMetadata.isMerged, DstBranch: pullRequestTargets[0].destinationReference, SrcBranch: pullRequestTargets[0].sourceReference, Author: authorArn}" \
+        --output text)
+
+        local dst_branch src_branch is_merge pull_id author title
+
+        IFS=$'\t'
+        read -r author dst_branch is_merge pull_id src_branch title <<< "${pull_info}"
```

Comment:
These two lines should be merged so the `IFS` variable is only affecting the `read` command

Reply:
Combined lines 197 and 198 to contain scope of IFS.